scripts/run.sh

```diff
@@ -17,8 +17,9 @@ run_cmd() {
 }
 
 do_build_tar() {
+  build_tar=$1
   if ([[ ${build_tar} == "True" && ! -f $TILE_TAR ]]) || [[ ${build_tar} == "Force" ]]; then
-    options="-c ${CONFIG_FILE} -v"
+    options="-c ${CONFIG_FILE} -v --overwrite"
```

Comment:
hm, but not we'd rebuild _every time_ the container is restarted, which might just be bcs of some config change. fine if you have andorra, but takes 20 mins if you have europe. can we at least do `--overwrite` only if `build_tar=Force`? that still wouldn't really work if you decided to build admins or smth else in the meantime though.. but IMO that was the real bug, if `build_tar=Force` it should overwrite regardless.